src/freenet/node/NodeStats.java

```diff
@@ -1459,6 +1465,7 @@ private String checkBandwidthLiability(double bandwidthAvailableOutputUpperLimit
 			PeerNode source, boolean isLocal, boolean isSSK, boolean isInsert, boolean isOfferReply, boolean hasInStore, int transfersPerInsert, boolean realTimeFlag, int maxOutputTransfers, int maxOutputTransfersPeerLimit, UIDTag tag) {
 		String name = input ? "Input" : "Output";
 		int peers = node.peers.countConnectedPeers();
+		boolean ignoreLocalVsRemoteBandwidthLiability = ignoreLocalVsRemoteBandwidthLiability();
```

Comment:
This duplication of names is confusing. Prepending `current` or something similar to the name would make its intended purpose more clear, and would allow the compiler to catch cases where one meant `ignoreLocalVsRemoteBandwidthLiability()` but wrote `ignoreLocalVsRemoteBandwidthLiability` (which are both of type `boolean`) or vice-versa.